src/components/cc-env-var-form/cc-env-var-form.js

```diff
@@ -40,17 +40,11 @@ export class CcEnvVarForm extends LitElement {
       addonName: { type: String, attribute: 'addon-name' },
       appName: { type: String, attribute: 'app-name' },
       context: { type: String, reflect: true },
-      error: { type: Boolean, reflect: true },
       heading: { type: String, reflect: true },
-      parserOptions: { type: Object, attribute: 'parser-options' },
       readonly: { type: Boolean, reflect: true },
       restartApp: { type: Boolean, attribute: 'restart-app' },
-      saving: { type: Boolean, reflect: true },
-      variables: { type: Array },
-      _currentVariables: { type: Array, state: true },
-      _description: { type: String, state: true },
-      _expertVariables: { type: Array, state: true },
-      _jsonVariables: { type: Array, state: true },
+      state: { type: Object },
+      _editorsState: { type: Object, state: true },
       _mode: { type: String, state: true },
       _isPristine: { type: Boolean, state: true },
```

Comment:
praise: the simplification feels nice, especially the `_expertVariables` and `_jsonVariables`